### ISSUE TITLE
Multiple presentation devices

### ISSUE THREAD

**Issue by obeletski** [w3c/presentation-api#97]
Media flinging use case requires Presentation API to support multiple presentation devices at the same time [1].
Specification has to be updated to reflect that in different sections:
* new requirement to be added
* methods for starting/joining session to support startup of a session on multiple screens
* screen selection algorithm and possibly UI to indicated that screen is already in use

[1] http://w3c.github.io/presentation-api/#use-cases-0

**Comment by obeletski** [comment]
Actually, two issue related the same subject already exist. https://github.com/w3c/presentation-api/issues/1 is closed now and https://github.com/w3c/presentation-api/issues/40 about the screen availability still open.

**Comment by anssiko** [comment]
The [proposed resolution][1] for #40 from the F2F seem to suggest no normative changes to the spec, if I'm not mistaken. The "Multiple presentations per controller" requirement was added in #95, and states "a single controller may be able to send content to multiple displays at once, and a single display may be able to host multiple presentations at once". We revised the terminology in #95, and controller is a shorthand for the controlling browsing context (earlier it was called opening browsing context).

@obeletski Do these clarifications address your issue? Feel free to clarify the flinging use case by submitting a proposal to the [Review the use cases and requirements][2].

[1]: https://github.com/w3c/presentation-api/issues/40#issuecomment-103814798
[2]: https://github.com/w3c/presentation-api/issues/68
